components/brave_welcome_ui/components/app.tsx

```diff
@@ -102,7 +98,12 @@ export class WelcomePage extends React.Component<Props, State> {
             <SlideContent>
               <WelcomeBox index={1} currentScreen={this.currentScreen} onClick={this.onClickLetsGo} />
               <ImportBox index={2} currentScreen={this.currentScreen} onClick={this.onClickImport} />
-              <SearchBox index={3} currentScreen={this.currentScreen} onClick={this.onClickConfirmDefaultSearchEngine} />
+              <SearchBox
+                index={3}
+                currentScreen={this.currentScreen}
+                onClick={this.onClickNext}
+                isDefaultSearchGoogle={true}
```

Comment:
@bsclifton @imptrx should this be hard-coded `true`? are there an API exposed to check whether or not G is default?

Reply:
I believe this is for me to populate - will take a look. This would be a matter of detecting whether you are in a region where Google is default (versus Qwant/DDG/etc)